### PR TITLE
community/goobook: move from testing, upgrade and add -doc

### PR DESCRIPTION
**Pull request by maxwell-k**

### CODE REVIEW

File: community/goobook/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Contributor: Ivan Tham <pickfire@riseup.net>
+# Maintainer: Ivan Tham <pickfire@riseup.net>
+pkgname=goobook
+pkgver=3.3
+pkgrel=0
+pkgdesc="Access Google contacts from the command line"
+url="https://gitlab.com/goobook/goobook"
+arch="noarch"
+license="GPL-3.0"
```
Comment (maxice8) [inline community/goobook/APKBUILD]: GPL-3.0 is not a valid spdx identifier

Comment (maxwell-k) [reply community/goobook/APKBUILD]: @maxice8 thank you! I have fixed this. It was sneaking through the sanity check so I have also filed https://github.com/alpinelinux/abuild/pull/84

**Comment by maxwell-k** [comment]
Amended in line with comment from @maxice8 

The line `cd "$builddir"` is required in a split function

**Comment by TBK** [comment]
My PR should be merged first https://github.com/alpinelinux/aports/pull/7606 and then this should be rebased.

**Pull request closed without merging**